client/components/theme-collection/index.tsx

```diff
@@ -101,13 +120,19 @@ export default function ThemeCollection( {
 					<div className="theme-collection__description">{ preventWidows( description ) }</div>
 				</div>
 				<div className="theme-collection__carousel-controls">
-					<Button className="theme-collection__see-all" onClick={ onSeeAll }>
+					<Button className="theme-collection__see-all" onClick={ onSeeAllAction }>
 						{ translate( 'See all' ) }
 					</Button>
-					<Button className="theme-collection__carousel-nav-button theme-collection__carousel-nav-button--previous">
+					<Button
```

Comment:
Should there be a specific event for when you hit the end and rewind the carousel? It kind of shows that people were interested in viewing more.

Maybe a nice to have.

Reply:
I've thought about that too, but I'm not sure about the business value of it. I've thought of both situations where we get high and low clicks on the back button, and what we could do with that data, but nothing meaningful came to mind. But if we think it's a good fit I could add it.

Reply:
I think it's enough to track the nav buttons themselves and "See all" to determine engagement. 🤔 
And I agree with Richard that clicking on "See all" should (ideally at least) directly answer the question "are people interested in viewing more?".